test/e2e/lib/pages/gutenboarding/domains-page.js

```diff
@@ -21,23 +21,39 @@ export default class DomainsPage extends AsyncBaseContainer {
 		);
 	}
 
+	async enterDomainQuery( query ) {
+		const searchFieldSelector = By.css( '.domain-picker__search input[type="text"]' );
+		await driverHelper.setWhenSettable( this.driver, searchFieldSelector, query );
+		// After typing the new query value, wait for domain suggestions to reload.
+		// The sleep value should be higher than the DOMAIN_SEARCH_DEBOUNCE_INTERVAL defined in domain-picker.
+		//https://github.com/Automattic/wp-calypso/blob/trunk/packages/domain-picker/src/constants.ts#L18
+		await this.driver.sleep( 500 );
+	}
+
+	async waitForDomainSuggestionsToLoad() {
+		const placeholderSelector = By.css( '.domain-picker__suggestion-item.placeholder' );
+		await driverHelper.waitTillNotPresent( this.driver, placeholderSelector );
```

Comment:
`waitTillNotPresent` can be a point of flakiness though, in this example here, https://github.com/Automattic/wp-calypso/pull/51863/files#diff-b7f8735b885b065dc18c161fbf8de3c5bfecafd5fd04319842a8eea5f5c32180L77, the spinner disappears too quickly so it got stuck there.

Reply:
Oh yeah right, `waitTillNotPresent` will just slide through if the element is not present. :) Okay, let's see what the E2E results show overtime.